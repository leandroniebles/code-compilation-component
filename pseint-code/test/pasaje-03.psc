subproceso val(a por valor)
FinSubProceso
Proceso sin_titulo
	Dimension w[10];
	val(w);
FinProceso
