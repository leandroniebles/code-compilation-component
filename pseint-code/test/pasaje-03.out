=== Line 5: ExeError 281
